spec/server.test.js

```diff
@@ -0,0 +1,13 @@
+const request = require('supertest');
+const app = require('./server/index.js');
+
+
+describe('get /items/:id', () => {
+  test('It should connect to the server', (done) => {
+    request(app).get('/items/3')
+      .then((response) => {
+        expect(response.statusCode).toBe(200);
+      });
+    done();
```

Comment:
Done should be called after the request has completed. This is being run immediately after the request is sent.